scripts/publish.js

```diff
@@ -81,6 +81,9 @@ if (/master/.test(currentBranch)) {
 
 // Create a git tag if we are publishing a specific version
 if (!/latest/.test(DOCKER_IMAGE_VERSION)) {
+  shell.exec(`echo ${DOCKER_IMAGE_VERSION} > version.txt`);
```

Comment:
Do you want to add version.txt to the .dockerignore? Or do you want that included as part of the docker image?

Reply:
let me add it to .dockerignore for now.